src/utils/association-record.js

```diff
@@ -0,0 +1,11 @@
+import orm from './orm';
+
+async function get(model, pk) {
+  const record = await orm.findRecord(model, pk);
+  if (!record) {
+    throw new Error(`related ${model.name} with pk ${pk} does not exist.`);
+  }
+  return record;
+}
+
+exports.get = get;
```

Comment:
in another file because will be needed for https://github.com/ForestAdmin/forest-express-sequelize/pull/585